docs/_Sidebar.md

```diff
@@ -1,29 +1,46 @@
-**Deployment Tutorials**
+## Deployment Tutorials
+
+
+### PowerAuth Server
```

Comment:
I am not sure whether we may use heading in the sidebar and what level. Is it somewhere rendered to check it visually?

Reply:
Study the following repository (Jekyll, Ruby, Liquid) for details:

https://github.com/wultra/wultra-developers

Reply:
> If you do not know if this is correct, what is reasoning in just changing it and doing it differently?

Because @romanstrobl asked for separation https://github.com/wultra/powerauth-server/pull/721#discussion_r893501203

Reply:
Please keep the **BOLD** approach, not headings. The headings are visible in the left bar in our developer portal, these should not be headings. Also, the developer portal is written to slice according to `<strong>` element.

Reply:
Got it, marking this conversation as resolved. I have already seen locally built developer portal.